extensions/VisualEditor/wikia/modules/ve/init/ve.init.wikia.Optimizely.js

```diff
@@ -0,0 +1,19 @@
+/*!
```

Comment:
Sure, I can change the name to getToolbarABTestVariantNumber.

It can also be moved to ve.utils if you want:
My reason for making a separate file is that there are more VE A/B tests upcoming (INT-328, INT-304, INT-307), so my intention was simply to group everything together in this new file as we will likely want to run several A/B tests at the same time.

Let me know if I should still move it.

Code formatting: This file failed the linter on my first commit, but the second commit should already have fixed it. Is it possible to run the linter locally before commiting?